spartan/bootstrap.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+source $(git rev-parse --show-toplevel)/ci3/source_bootstrap
+
+cmd=${1:-}
+
+hash=$(cache_content_hash \
+  ../spartan/.rebuild_patterns)
```

Comment:
Why not just . rebuild_patterns
This can reference yarn project and be the only call in the CI.yml

Reply:
Will update